build/build.sln

```diff
@@ -777,6 +773,12 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "AddressBook", "AddressBook"
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "AddressBook.UnitTests", "Tests\UnitTests\AddressBook\AddressBook.UnitTests.vcxproj", "{62E53898-65C2-4401-BF58-FBFB728E1B27}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "3rdParty", "3rdParty", "{DA66E27D-9C75-423E-BC9A-78D380FDB99D}"
```

Comment:
Agreed and because that part is WIP this is a temp place to catch any build regressions. It will move out as part of Meredith's work

---
In reply to: [74129301](https://github.com/Microsoft/WinObjC/pull/853#discussion_r74129301) [](ancestors = 74129301)